app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -114,14 +115,17 @@ data class HomeState(
 
 // TODO change to extend [SnackBarMessage]
 sealed class HomeSnackbarState {
+    object None : HomeSnackbarState()
+    data class ClearConversationContentSuccess(val isGroup: Boolean) : HomeSnackbarState()
+    data class ClearConversationContentFailure(val isGroup: Boolean) : HomeSnackbarState()
+
     class SuccessConnectionIgnoreRequest(val userName: String) : HomeSnackbarState()
     object MutingOperationError : HomeSnackbarState()
     object BlockingUserOperationError : HomeSnackbarState()
-    class BlockingUserOperationSuccess(val userName: String) : HomeSnackbarState()
+    data class BlockingUserOperationSuccess(val userName: String) : HomeSnackbarState()
```

Comment:
Any reason declaring these 2 classes as `data classes` but not for `SuccessConnectionIgnoreRequest` ?

Reply:
no idea I was not responsible for the other ones